wings/order_book.pyx

```diff
@@ -296,12 +296,22 @@ cdef class OrderBook(PubSub):
                 total_cost += order_book_row.amount * order_book_row.price
                 total_volume += order_book_row.amount
                 if total_volume >= volume:
+                    incremental_amount = total_volume - volume
```

Comment:
can you explain this change?

Reply:
sure, this is to make it more accurate. Say you want to buy 5000 of a coin via market order, Ask Orderbook is like this: 
4000 : 0.1
500 : 0.2
400: 0.3
5000 : 0.4

Now the way the earlier code was written it would do a weighted average of this entire orderbook which would make your cost to be higher (2620/9900) vs ( 660/5000). Now as you are buying 5000 instead of 9900 the latter is more accurate.